### PR TITLE
Added Project section

### PR DESCRIPTION
**Pull request by prishamehta01**
This PR adds a new Projects section to the portfolio website, complete with responsive styles for project items, icons, and buttons. The goal is to enhance the visual appeal and usability of the portfolio.
Created a new .projectsC class to style the projects section.
Added styles for project items, including hover effects and spacing.

Thank You.

### CODE REVIEW

**Comment by prishamehta01** [comment]
@divyanshudhruv Please check this out!!

**Comment by divyanshudhruv** [comment]
@prishamehta01 sure.

**Comment by divyanshudhruv** [comment]
@prishamehta01 actually there are some issues in the code you provided. The fonts are not of the same font-family as the others are, and the project section has a bit of a small height. Please fix those(add the font-family to the text and increase the height of the projects section) , and I'll be merging it soon after reviewing it. Don't worry, you will get hacktoberfest-accepted label soon. 😄😄

File: css/style.css [hunk 1]
```diff
@@ -401,89 +401,83 @@ a:visited {
...
-  bottom: 0;
+  flex-direction: column; /* Stack projects vertically */
   align-items: center;
   justify-content: center;
-  width: 100%;
-  height: 200px;
-  background-color: none;
-} */
+  margin-top: 20px;
+  gap: 20px; /* Space between project items */
```
Comment (divyanshudhruv) [inline css/style.css:438]: you can add height: ... here

File: css/style.css [hunk 2]
```diff
@@ -401,89 +401,83 @@ a:visited {
...
+}
+
+.projectText {
+  flex-grow: 1; /* Take remaining space */
+}
+
+.projectText .text {
+  font-weight: 500;
+  font-size: 24px;
+  color: #272341;
```
Comment (divyanshudhruv) [inline css/style.css:469]: you should either add font-family:"Montserrat" or font-family:"Poppins". Try using both and select which looks the best

File: css/style.css [hunk 3]
```diff
@@ -401,89 +401,83 @@ a:visited {
...
+}
+
+.projectText .textMain {
+  font-size: 16px;
+  color: #666;
```
Comment (divyanshudhruv) [inline css/style.css:474]: you should add font-family:"Montserrat" or font-family:"Poppins" and then select which looks the best

File: css/style.css [hunk 4]
```diff
@@ -401,89 +401,83 @@ a:visited {
...
+}
+
+.projectItem .btn {
+  margin-left: 20px; /* Space between text and button */
```
Comment (divyanshudhruv) [inline css/style.css:478]: you should add font-family:"Montserrat" or font-family:"Poppins" and then select which looks the best...

**Review by divyanshudhruv — COMMENTED** [review]
please fix these. Good luck

**Comment by prishamehta01** [comment]
@divyanshudhruv I've resolved all the issues
Everything looks uniformed now!!
Thank you so much for your guidance😊

**Comment by prishamehta01** [comment]
Heyy!! Can we please move forward 🥺 I have lots to do..

**Comment by divyanshudhruv** [comment]
@prishamehta01 sorry for such a long time in merging it. I'll merge this PR under 2hrs. Thank you for the contribution

**Comment by prishamehta01** [comment]
No problem, I understand.
Take you time
Thank you😄

**Review by divyanshudhruv — APPROVED** [review]
seems nice

**Pull request merged**